firefox-ios/Client/Frontend/Onboarding/Protocols/OnboardingCardViewController.swift

```diff
@@ -69,8 +68,7 @@ class OnboardingCardViewController: UIViewController, Themeable {
         label.numberOfLines = 0
         label.textAlignment = .center
         let fontSize = self.shouldUseSmallDeviceLayout ? SharedUX.smallTitleFontSize : SharedUX.titleFontSize
-        label.font = DefaultDynamicFontHelper.preferredBoldFont(withTextStyle: .largeTitle,
-                                                                size: fontSize)
+        label.font = FXFontStyles.Bold.largeTitle.scaledFont().withSize(fontSize)
```

Comment:
@bmihai23 As part of our standardizing fonts initiative, we would like to move away from specifying `fontSize` and rely on the design system fonts. This ticket is a tricky case because our font size varies for different device layouts. 

To make minimal changes, I would suggest we find the font styles that matches the font size and replace them. Let me know if you have any questions or what you think about this proposal.

Reply:
Hi @cyndichin, I've made the required changes to standardize fonts and updated the screenshots. Please let me know if anything else is needed. Thanks 🙏

Reply:
thanks @bmihai23! This is exactly what I was thinking, looks great!